### PR TITLE
More CWL work

### PR DESCRIPTION
**Pull request by gijzelaerr**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,11 +43,35 @@ data/${FILE}:
...
+		--fluxcal data/small/00/raw/3C295.MS \
+		--polcal data/small/00/raw/3C138.MS \
+		--target data/small/00/raw/NGC807.MS
+
+cwl-crosscal: $(VENV3)/bin/cwltool data/small
+	$(CWLTOOL) --debug cwl/steps/ccal.cwl \
+		--fluxcal data/small/00/raw/3C295.MS \
+		--polcal data/small/00/raw/3C138.MS \
+		--target data/small/00/raw/NGC807.MS
+
```
Comment (tammojan) [inline Makefile]: How is this going to work with multiple beams (i.e. more than just `'00'`)? Will it work when `polcal=''`, `flluxcal=''`?

File: apercal/modules/convert.py [hunk 2]
```diff
@@ -339,15 +342,13 @@ def ms2miriad(self):
...
                 logger.info('Converting target beam datasets from UVFITS to MIRIAD format.')
                 if self.convert_targetbeams == 'all':
-                    datasets = glob.glob(mspath_to_fitspath(self.get_crosscalsubdir_path('[0-9][0-9]'),
-                                                            self.get_target_path()))
+                    datasets = self.get_datasets()
                     logger.debug('Converting all available target beam datasets')
                 else:
                     beams = self.convert_targetbeams.split(",")
-                    datasets = [mspath_to_fitspath(self.get_crosscalsubdir_path(str(b).zfill(2)), self.target) for b in beams]
+                    datasets = self.get_datasets(beams)
```
Comment (tammojan) [inline apercal/modules/convert.py]: This should be a list of fits paths, I think get_datasets gets MS paths. Probably you mean `[mspath_to_fitspath(mspath) for mspath in self.get_datasets(beams)]`

File: apercal/modules/preflag.py [hunk 3]
```diff
@@ -441,7 +441,7 @@ def manualflag_auto(self):
                     logger.debug('Flagging auto-correlations for all target beams')
                 else:
                     beams = self.preflag_manualflag_targetbeams.split(",")
-                    datasets = [self.get_datasets(beams=beams)]
```
Comment (tammojan) [inline apercal/modules/preflag.py]: Has this ever worked? Seems like it should be broken on master?

File: Makefile (hunk 1 above)
Comment (gijzelaerr) [reply Makefile]: this is just a makefile shortcut to test run this CWL file outside a container.

**Pull request merged**